docs/en/latest/dns.md

```diff
@@ -74,3 +74,60 @@ If you want to specify the port for the upstream server, you can add it to the `
     "type": "roundrobin"
 }
 ```
+
+Another way to do it is via the SRV record, see below.
+
+### SRV record
+
+By using SRV record you can specify the port and the weight of a service.
+
+Assumed you have the SRV record like this:
+
+```
+; under the section of blah.service
+A       300 IN      A     1.1.1.1
+B       300 IN      A     1.1.1.2
+B       300 IN      A     1.1.1.3
+srv   86400 IN    SRV 10       60     1980 A
+srv   86400 IN    SRV 10       20     1981 B
+```
+
+Upstream configuration like:
+
+```json
+{
+    "id": 1,
+    "discovery_type": "dns",
+    "service_name": "srv.blah.service",
+    "type": "roundrobin"
+}
+```
+
+is the same as:
+
+```json
+{
+    "id": 1,
+    "type": "roundrobin",
+    "nodes": {
```

Comment:
should use `[` here, it an array object here.

Reply:
@membphis 
Solved.